src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -877,6 +877,17 @@ public bool NodeAutocompleteMachineLearningIsChecked
             }
         }
 
+        /// <summary>
+        /// Controls if the the Node autocomplete Machine Learning option is beta from feature flag
+        /// </summary>
+        public bool NodeAutocompleteMachineLearningIsBeta
+        {
+            get
+            {
+                return DynamoModel.FeatureFlags.CheckFeatureFlag("NodeAutocompleteMachineLearningIsBeta", false);
```

Comment:
@mjkkirschner @sm6srw @pinzart Let me know if I used this correctly. I think I will have to  register this flag as well

Reply:
I would register and test that the flag is working in "dev" (debug builds) before merging this PR - as one issue with the feature flags manager is that it's uses another process and tries to avoid blocking startup so timing is important depending on where you use the flag - I see in this case you are not using the feature flag event - 
`FlagsRetrieved` - so it's possible you will *always* be checking that flag before it is retrieved.

Reply:
I see, @mjkkirschner The reason I do not use events is that this getter will not be triggered until the user opens up the Preferences panel which pretty much always happens after DynamoModel is initialized. But maybe I am wrong because FeatureFlags is more like a singleton?

Reply:
@QilongTang that sounds like safe usage, but I think it's worth checking if you can - I can help you create the flag if you want?

Reply:
@mjkkirschner Sure, currently the beta tag will be hidden. Would you create the tag and give it `true` value? We need to see to true anyway for Global Launch so need it created anyway :)

Reply:
will do so now.

Reply:
![Screen Shot 2022-12-02 at 10 16 25 AM](https://user-images.githubusercontent.com/508936/205325310-22c7c12e-c5c8-4479-b963-d7192f185a41.png)

Reply:
I cannot set it true in prod, only owners can do that.

Reply:
@mjkkirschner Would you remind me how to use the feature flag none-prod server?

Reply:
a debug build will check "dev", release builds will check "prod"

Reply:
It works
<img width="665" alt="image" src="https://user-images.githubusercontent.com/3942418/205347274-76089124-3fa7-40ad-a724-ff08875ddeaf.png">